WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -110,17 +110,21 @@ struct OrderPaymentSection: View {
                     }
                 }
 
+            appliedGiftCardsSection
+
             taxesSection
                 .fullScreenCover(isPresented: $shouldShowTaxEducationalDialog) {
                     TaxEducationalDialogView(viewModel: viewModel.taxEducationalDialogViewModel,
                                              onDismissWpAdminWebView: viewModel.onDismissWpAdminWebViewClosure)
                         .background(FullScreenCoverClearBackgroundView())
                     }
 
-            TitleAndValueRow(title: Localization.discountTotal, value: .content(viewModel.discountTotal))
-                .renderedIf(viewModel.shouldShowDiscountTotal)
+            VStack(alignment: .leading, spacing: .zero) {
```

Comment:
Note for the reviewer: the discount and order total rows are now in a VStack because the number of elements in the original VStack is reaching 10 with the gift card work. When there are more than 10 elements in the VStack, a not-so-obvious build error is shown.